.github/renovate.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": [
+    "local>hmcts/.github:renovate-config",
```

Comment:
could u plz create a separate PR and merge this in as this PR will take time to test and deploy ?